packages/azure-themes/src/azure/AzureThemeDark.ts

```diff
@@ -78,10 +78,14 @@ export const AzureThemeDark: ITheme = createTheme({
   },
   palette: {
     themePrimary: DarkSemanticColors.controlOutlines.accent,
+    neutralPrimary: DarkSemanticColors.text.body,
     neutralDark: DarkSemanticColors.text.body,
-    neutralLight: DarkSemanticColors.item.hover, // shimmer elements
+    neutralLight: CommonSemanticColors.dividers.lineSeparator, //shimmer and document card
     neutralLighter: DarkSemanticColors.item.hover, // shimmer elements
```

Comment:
just curious what are the shimmer elements look like))) can you share a screenshot on Teams? 😄

Reply:
You should be able to use the pr links I think! :)

http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/7681/merge/fabric-website-resources/dist/index.html#/examples/shimmer